generate/input/descriptor.json

```diff
@@ -71,11 +71,8 @@
   },
   "types":
   {
-    "annotated": {
+    "annotated_commit": {
```

Comment:
Can you remove this section completely? No use in having an empty section for this.